web/src/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import './styles/global.css';
+
+import Register from './components/register/register';
+import GroupView from './components/groupView/groupView';
+import Wave from './components/wave/wave';
+import './App.css';
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    
+    this.state = {
+      groupInfo: null
+    }
+    
+    this.handleUserData = this.handleUserData.bind(this)
+  }
+  
+  handleUserData(data) {
+    fetch("/group/"+data.groupId)
+      .then(response => response.json())
+      .then(res => {
+        if(res.message) {
+          alert("Unfortunately there was an error, please try again later.")
+        } else {
+          this.setState({
+            groupInfo: res
+          })
+        }
+      })
+      .catch(error => alert("Unfortunately there was an error, please try again later."))
+  }
+  
+  render() { 
+    return (
+      <div className="App">
+        <header className="App-header">
+          
+        </header>
+        <Wave />
+        <div className="App-body">
+          <div className="card">
+            { !this.state.groupInfo ? (<Register onUserDate={ this.handleUserData } />) : (<GroupView groupInfo={ this.state.groupInfo } />)}
+          </div> 
+        </div>
+        <footer className="App-footer">
+          <p>UI made with 💚 by Jan-Niklas</p>
```

Comment:
No offense, but I don't think this belongs in a group project

Reply:
Why not? I see no problem in showing who did which part of the project.

Reply:
This project will be used and developed by other people in the future, and generally this project is owned by the FSR/ESA. You should look at this like a project you work on at your job. You would not personally sign a website created for your job either.

Reply:
In fact, we do that often at work. I'll take it out even if the ESA team didn't see any problem with that.

Reply:
if you wanted to immortalize yourself, maybe you should keep it hidden. :racehorse: